package.json

```
@@ -43,8 +43,8 @@
     "@mikro-orm/core": "^5.0.0",
     "@mikro-orm/knex": "^5.0.0",
     "@mikro-orm/mongodb": "^5.0.0",
-    "@nestjs/common": "^8.0.0",
-    "@nestjs/core": "^8.0.0"
+    "@nestjs/common": "^9.0.0",
+    "@nestjs/core": "^9.0.0"
```

this would be breaking change, if there is no need to change anything at our place, lets allow v8 too and release this as minor instead
```suggestion
    "@nestjs/common": "^8.0.0 || ^9.0.0",
    "@nestjs/core": "^8.0.0 || ^9.0.0"
```